### DISCUSSION TITLE
include検索パスの指定

### DISCUSSION THREAD

**Discussion by SuperTurboZ** [AILight/AILZ80ASM#203] [Q&A]
includeのファイルを検索するパスを指定することは可能でしょうか？

include "../lib/hoge.inc"

の位置にあるファイルをコマンドライン等で事前に"../"にパスを通しておいて、ソースには

include "lib/hoge.inc"

と記述したいのですが、readmeやhelpに指定方法を見つけることができませんでした。

**Comment by AILight** [comment]
includeに対してではないのですが、アセンブルを行うカレントディレクトリを設定する事は可能です。 -cd のオプションを使っていただけると変える事が可能です。それ以外で、includeのパスを変更するにはちょっとオプション等を検討する必要があるかと思います。

**Comment by SuperTurboZ** [comment]
ご回答ありがとうございます。
現状では検索パスを指定する機能はないのですね。
スマートではないですが、事前にローカル下のlibフォルダへ必要なインクルードファイルをコピーしてからアセンブラを実行するなどの処理をすればやりたいことはできそうです。

**Reply by AILight** [reply]
ご要望の対応が遅くなり申し訳ございません。現在修正作業中ですが、include用の検索パスを指定できるコマンドラインオプションを追加させていただきます。「-ips ./lib1」のように検索ディレクトリを指摘できるようにしたいと考えております。

基本的な修正作業は完了しておりますので、問題がなければ次回のリリースからご利用になれると思います。
とても便利になる要望を頂きましてありがとうございました。引き続きよろしくお願いいたします。

**Comment by AILight** [comment]
対応版をリリースしました。
